src/main/kotlin/com/example/iot_management_device/controller/DeviceController.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.iot_management_device.controller
+
+import com.example.iot_management_device.dto.device.request.DeviceRequestDto
+import com.example.iot_management_device.dto.device.response.DeviceResponseDto
+import com.example.iot_management_device.dto.device.request.DeviceUpdateRequestDto
+import com.example.iot_management_device.service.device.DeviceService
+import jakarta.validation.Valid
+import org.springframework.http.HttpStatus
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/device")
```

Comment:
try to use the same api style in the whole project (like plural nouns for the whole project, not `/device` and `/users` in the same project)